src/components/AppElements/components/CodeEditor/hooks/htmlImportPluginHooks.js

```diff
@@ -0,0 +1,26 @@
+export default function () {
+	return `const htmlImportPluginHooks = {
+		cancel: () => {
+			return new Promise(resolve => {
+				importInstance.hide();
+				resolve();
+			});
+		},
+		close: () => {
+			return new Promise(resolve => {
+				importInstance.hide();
+				resolve();
+			});
+		},
+		onReplaceImages: (imageSrcArray) => {
+			return new Promise(resolve => {
+				resolve(imageSrcArrayReplaced);
```

Comment:
```suggestion
				resolve(imageSrcArrayReplaced);
```
imageSrcArrayReplaced nincs bedefiniálva sehol, úgyhogy vagy töröld ki vagy csinálj egy map example-t ami meggcsinálja az input-ból az output-ot csere nélkül